packages/manager/apps/identity-access-management/src/pages/tagManager/components/tagsListTopBar/tagsListTopbar.component.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import AssignTagButton from '@/components/assignTagsButton/assignTagsButton';
+import TagsListFilter, {
+  TagsListFilterProps,
+} from '../tagsListFilter/tagsListFilter.component';
+
+export type TagsListTopbarProps = TagsListFilterProps;
+
+export default function TagsListTopbar({
+  isShowSystemChecked,
+  setIsShowSystemChecked,
+  isShowUnassignedResourcesChecked,
+  setIsShowUnassignedResourcesChecked,
+}: TagsListTopbarProps) {
+  const assignTag = () => {
+    console.log('TODO: Assign tag');
+  };
+
+  return (
+    <div className="flex justify-between w-full items-center mr-4">
+      <AssignTagButton onClick={assignTag} />
+      <TagsListFilter
+        isShowUnassignedResourcesChecked={isShowUnassignedResourcesChecked}
+        setIsShowUnassignedResourcesChecked={
+          setIsShowUnassignedResourcesChecked
+        }
```

Comment:
Would be defined in the context but should be
```suggestion
        onClickUnassignedResourcesChecked={ () =>
          setIsShowUnassignedResourcesChecked(value=> !value)
        }
```